plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -418,20 +418,25 @@ function OPDSBrowser:genItemTableFromCatalog(catalog, item_url)
                         -- https://vaemendis.net/opds-pse/
                         -- «count» MUST provide the number of pages of the document
                         -- namespace may be not "pse"
-                        local count
+                        local count, last_read
                         for k, v in pairs(link) do
                             if k:sub(-6) == ":count" then
                                 count = tonumber(v)
-                                break
+                            elseif k:sub(-9) == ":lastRead" then
+                                last_read = tonumber(v)
                             end
                         end
                         if count then
-                            table.insert(item.acquisitions, {
+                            local acquisition = {
                                 type  = link.type,
                                 href  = link_href,
                                 title = link.title,
                                 count = count,
-                            })
+                            }
+                            if last_read and last_read > 0 then
+                                acquisition.last_read = last_read
+                            end
+                            table.insert(item.acquisitions, acquisition)
```

Comment:
This is just one additional line in the table:
`last_read = last_read and last_read > 0 and last_read or nil,`